src/Paramore.Brighter/Subscription.cs

```diff
@@ -117,7 +117,15 @@ public class Subscription
         /// or busy
         /// </summary>
         /// <value>The timeout in miliseconds.</value>
-        public int TimeoutInMiliseconds { get; }
+        [Obsolete("Use TimeoutInMilliseconds")]
+        public int TimeoutInMiliseconds { get => TimeoutInMilliseconds; }
+
+        /// <summary>
+        /// Gets the timeout in milliseconds that we use to infer that nothing could be read from the channel i.e. is empty
+        /// or busy
+        /// </summary>
+        /// <value>The timeout in mi;liseconds.</value>
+        public int TimeoutInMilliseconds { get; }
```

Comment:
Because this is a public method on a class exposed to consumers, this forms a breaking change, and we would need to version. To fix this, you need a new property with the correct name, and you need the old property to use that new property for it's implementation. Then you need to mark the old property as obsolete, note that folks should use the correctly spelled one and indicate that we will switch over in v10

Reply:
thats what i did

Reply:
Not sure how I missed that :-D

Well, I guess you did it our preferred way then...